christfellowshipchurch/web-app-v2
# Change "Need Help" to "Help Me Find a Group"

### About
This PR changes "Need Help" to "Help Me Find a Group" on the Group Finder

### Test Instructions
On the Group Finder check the following locations to see the changes:
  1.  At the bottom of the page for each HUB
  2. Button at the top of each page

### Screenshots
<img width="1426" alt="Screen Shot 2021-09-28 at 3 05 11 PM" src="https://user-images.githubusercontent.com/46769629/135150379-ff854266-f6ec-4c84-9839-7c1cdbb99e27.png">

<img width="1433" alt="Screen Shot 2021-09-28 at 3 05 25 PM" src="https://user-images.githubusercontent.com/46769629/135150386-f05ce00e-9ff9-4dd9-8e95-2314ceb9dcc7.png">

### Closes Tickets
[CFDP-1699](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-1699)

### Related Tickets
N/A
